rdmd.d

```diff
@@ -59,6 +59,15 @@ version(unittest) {} else
 int main(string[] args)
 {
     //writeln("Invoked with: ", args);
+    string binPath = thisExePath();
+    if (binPath.exists && binPath.isFile && filenameCmp(baseName(binPath),
+                                                        baseName(args[0])) == 0)
```

Comment:
Imho they aren't necessary, just confuse the reader and prevent symlinks, so I would suggest to remove them.
What do you think @CyberShadow?